src/components/dashboard/tracks/TrackList.js

```diff
@@ -35,15 +38,38 @@ function TracksList() {
       link: '/dashboard/track/devops',
     },
   ];
+  const { items } = data;
+
+  useEffect(() => {
+    getTracksAction();
+  }, []);
```

Comment:
Codacy found an issue: [React Hook useEffect has a missing dependency: 'getTracksAction'. Either include it or remove the dependency array. If 'getTracksAction' changes too often, find the parent component that defines it and wrap that definition in useCallback.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5801772)